events/event.h

```diff
@@ -42,6 +42,7 @@ namespace QMatrixClient
                 RoomName = RoomStateEventBase + 1,
                 RoomAliases, RoomCanonicalAlias, RoomMember, RoomTopic,
                 RoomAvatar, RoomEncryption,
+                CallInvite, CallCandidates, CallAnswer, CallHangup,
```

Comment:
I think it makes sense to use another bit (pick your favourite from the first byte) to indicate that an event is a call event. Doesn't change much so far; but clients might want to check if an event is a call event to adorn it with a relevant icon, e.g. See the code for `Event::isStateEvent()`. On a related note, you should move the new events to the room non-state events block (before `RoomStateEventBase`).